packages/calcite-components/src/components/interfaces.ts

```diff
@@ -28,7 +29,7 @@ export type SelectionMode =
   | "multiple";
 export type Scale = "s" | "m" | "l";
 export type Status = "invalid" | "valid" | "idle";
-export type Width = "auto" | "half" | "full";
+export type Width = "s" | "m" | "l" | "auto" | "half" | "full";
```

Comment:
@Erik suggested to do export type `Width = Scale & "auto" | "half" | "full"`; Same for Height above. Height would just be export type `Height = Scale`. 

I felt like it might make things a bit confusing to read. Height, Width, and Scale are different props even though they have an overlap in type (with type Scale). s/m/l for scale does something else for width/height. Should they just have their own type names? It hurts my eyes to read type Height = Scale :)

Erik's suggestion is purely for DRY reasons. It's really just a union type to keep from having to define "s" | "m" | "l" in 3 different places. type Height = Scale, does look weird, but it has more semantics at least. 

WDYT @jcfranco

Reply:
I see your point about type `type Height = Scale` being potentially confusing, but I think it’s fine given the consolidation of `widthScale` and `heightScale` props.